spec/motion-spec/matchers/respond_to_spec.rb

```diff
@@ -0,0 +1,34 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::RespondTo" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.